src/EnergyPlus/DaylightingManager.cc

```diff
@@ -54,9 +54,8 @@
 // ObjexxFCL Headers
 #include <ObjexxFCL/Array.functions.hh>
 #include <ObjexxFCL/Fmath.hh>
-#include <ObjexxFCL/Vector2.hh>
 #include <ObjexxFCL/Vector3.hh>
-#include <ObjexxFCL/Vector4.hh>
+// #include <ObjexxFCL/Vector4.hh>
```

Comment:
Need to remove this. I believe that after this, there is only one remaining use of `Vector4` anywhere.

Reply:
Wooooo!